### PR TITLE
PSMR sirf_exercises (Synergistic)

### PR DESCRIPTION
**Pull request by samdporter**

### CODE REVIEW

**Comment by danieldeidda** [comment]
@samdporter Did you close this by mistake?

**Comment by danieldeidda** [comment]
there is a conflict for SPECT_acqusition_model I am guessing you have an older version

File: notebooks/Synergistic/cil_joint_tv_PET.ipynb [hunk 1]
```diff
@@ -60,9 +60,51 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 3,
```
Comment (danieldeidda) [inline notebooks/Synergistic/cil_joint_tv_PET.ipynb:63]: count should be null

**Comment by samdporter** [comment]
Sorry, Daniel. This was supposed to be closed

**Pull request closed without merging**